.github/workflows/skaffold/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM gcr.io/k8s-skaffold/skaffold:v1.3.1
```

Comment:
is this skaffold action still needed?

Reply:
Nope, forgot to remove that after adding goreleaser 👍